content/en/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md

```diff
@@ -119,8 +119,8 @@ Here are some of the images we built：
 - `gcr.io/kubernetes-e2e-test-images/volume/iscsi:2.0`
 - `gcr.io/kubernetes-e2e-test-images/volume/nfs:1.0`
 - `gcr.io/kubernetes-e2e-test-images/volume/rbd:1.0.1`
-- `k8s.gcr.io/etcd:3.3.15`
-- `k8s.gcr.io/pause:3.1`
+- `registry.k8s.io/etcd:3.3.15` (Previously it was `k8s.gcr.io/etcd:3.3.15`, updated on 2023-03-20.)
+- `registry.k8s.io/pause:3.1` (Previously it was `k8s.gcr.io/pause:3.1`, updated on 2023-03-20.)
```

Comment:
I would leave the text in this article exactly as it was.

Reply:
Alright! I reverted the changes, Thanks.